Superposition/Tasks.qs

```diff
@@ -118,31 +118,34 @@ namespace Quantum.Kata.Superposition {
 
         // ...
     }
-
-
+    
+    
     // Task 9. Superposition of all basis vectors
     // Input: N qubits in |0...0⟩ state.
     // Goal: create an equal superposition of all basis vectors from |0...0⟩ to |1...1⟩
     // (i.e. state (|0...0⟩ + ... + |1...1⟩) / sqrt(2^N) ).
     operation AllBasisVectorsSuperposition (qs : Qubit[]) : Unit {
         // ...
     }
+    
+    
+	// Task 10. Superposition of all even numbers
+    // Input: N qubits in |0...0⟩ state.
+	// Goal: create a superposition of all even numbers on n qubits
+	operation EvenNumbersSuperposition (qs : Qubit[]) : Unit is Adj {
```

Comment:
I would suggest adding a little twist to this task to make it more interesting: add an extra parameter `odd : Bool` and specify that if it is true you need to prepare a superposition of all odd numbers, and if it is false - of all even numbers. The difference in the reference implementation will be just applying an X gate to qs[N-1] if odd is true (which corresponds to flipping the least significant bit), but it will make it interestingly different from the previous task.